### ISSUE TITLE
information_schema.columns is not updated when dropping table column

### ISSUE THREAD

**Issue by fflorent** [oguimbal/pg-mem#96]
## Context

When I drop a column from a table and try to inspect the columns through information_schema.columns, the dropped columns are still present.

## Steps to reproduce

1. Go to: https://oguimbal.github.io/pg-mem-playground/
2. Evaluate the below instructions:
```sql
CREATE TABLE foo (
    uuid   VARCHAR NOT NULL
);

SELECT * from information_schema.columns where table_name='foo'; -- This shows uuid as expected

ALTER TABLE foo DROP COLUMN uuid;

SELECT * from information_schema.columns where table_name='foo'; -- This should not show uuid anymore, but it does
```

I ran the above instructions in postgresql, it didn't show any column after the `ALTER TABLE - DROP COLUMN` instruction

---

Thanks for this lib :pray:

**Comment by oguimbal** [comment]
Hi !

Thanks for the detailed report. 
That's a problem indeed... I'll try to fix that today :)

Disclaimer: things in `information_schema` are highly experimental... I implemented them `partially`, mostly because ORMs need them.

**Comment by fflorent** [comment]
@oguimbal Thanks for your reply. I have just proposed a PR for solving this issue :).

**Comment by oguimbal** [comment]
Fixed in `pg-mem@1.9.4`
